errors.js

```diff
@@ -242,6 +242,12 @@ Errors.InvalidHeaderTypeError = TypedError({
     name: null
 });
 
+Errors.InvalidInitHostPortError = TypedError({
+    type: 'tchannel.invalid-init-host-port',
+    message: 'invalid host:port string in init header; got: {hostPort}',
```

Comment:
we should talk about cardinality of error strings at some point.

I think there is value in it because the lack of meta data in error frames and high cardinality string in error frame is the "best" user experience we can give our customers.